inc/class-wpseo-admin-bar-menu.php

```diff
@@ -127,10 +141,54 @@ public function add_menu( WP_Admin_Bar $wp_admin_bar ) {
 		 */
 		do_action( 'wpseo_add_adminbar_submenu', $wp_admin_bar, self::MENU_IDENTIFIER );
 
-		$this->add_keyword_research_submenu( $wp_admin_bar );
-
 		if ( ! is_admin() ) {
+			$indexable     = $this->indexable_repository->for_current_page();
+			$focus_keyword = ( is_null( $indexable->primary_focus_keyword ) ) ? __( 'not set', 'wordpress-seo' ) : $indexable->primary_focus_keyword;
+
+			if ( is_singular() || is_tag() || is_tax() || is_category() ) {
+				$wp_admin_bar->add_menu(
+					[
+						'parent' => self::MENU_IDENTIFIER,
+						'id'     => 'wpseo-seo-focus-keyword',
+						'title'  => __( 'Focus keyword: ', 'wordpress-seo' ) . '<span class="wpseo-focus-keyword">' . $focus_keyword . '</span>',
+						'meta'   => [ 'tabindex' => '0' ],
+					]
+				);
+
+				$wp_admin_bar->add_menu(
+					[
+						'parent' => self::MENU_IDENTIFIER,
+						'id'     => 'wpseo-seo-score',
+						'title'  => 'SEO Score: ' . $this->score_icon_helper->for_seo( $indexable, 'adminbar-sub-menu-score' )->present(),
```

Comment:
I believe this should be translatable, as we do it in other places in the plugin.

Same with Readability.